plugins/org.jboss.reddeer.junit.extension/src/org/jboss/reddeer/junit/extension/before/test/impl/DoNotDownloadMavenIndexesExt.java

```diff
@@ -98,4 +98,9 @@ private void disableMavenDownloadRepoIndexOnStartup() {
 	public boolean hasToRun() {
 		return DoNotDownloadMavenIndexesExt.DISABLE_MAVEN_DOWNLOAD_REPO_INDEX;
 	}
+
+	@Override
+	public long getPriority() {
+		return 1000;
```

Comment:
All constants should be be defined as constants (public static final). Also update the JavaDoc so that it is clear what's the priority.